zeppelin-interpreter/src/main/java/org/apache/zeppelin/helium/HeliumPackage.java

```diff
@@ -18,13 +18,16 @@
 
 import org.apache.zeppelin.annotation.Experimental;
 
+import java.util.Map;
+
 /**
  * Helium package definition
  */
 @Experimental
 public class HeliumPackage {
   private HeliumType type;
   private String name;           // user friendly name of this application
+  private String version;
```

Comment:
'artifact' field usually includes version information of the artifact.
I think 'version' field duplicates information included in 'artifact' field.

What do you think? can you elaborate reason for adding version field?